test/blackbox-tests/test-cases/dune-project-meta/github11205.t

```diff
@@ -0,0 +1,66 @@
+Support for negation in package dependencies:
```

Comment:
Given this is a new feature, maybe this test could be added in some other existing cram test that is testing the other operators?

My concern is that `github11205` is not very descriptive and if we need to add a test on some specific behavior we might forget that this already exists and then duplicate it.

Reply:
I moved the negation tests to the existing `binops.t` and renamed it `operators.t`